CONTRIBUTING.md

```diff
@@ -110,3 +110,103 @@ If you need to submit a patch that changes only data but does not alter the sche
 
 If in the future we discover a use-case where a python patch must be applied for which there *is no corresponding SQL patch*, then a blank SQL patch file will still need to created.
+
+##SQL coding guidelines
+Since the `qiita_db` code contains a mixture of python code and SQL code, here are some coding guidelines to add the SQL code to Qiita:
+
+1. Any SQL keyword should be written uppercased:
+  * Wrong:
+  ```python
+  sql = "select * from qiita.qiita_user"
+  ```
+  * Correct:
+  ```python
+  sql = "SELECT * FROM qiita.qiita_user"
+  ```
+2. Triple quotes are preferred for the SQL statements, unless the statement fits in a single line, since most editors and GitHub will detect SQL statement and apply highlighting accordingly. See point 3 for logical grouping of the SQL statements and indentation:
+  * Wrong:
+  ```python
+  sql = ("SELECT processed_data_status FROM qiita.processed_data_status pds JOIN "
+         "qiita.processed_data pd USING (processed_data_status_id) JOIN "
+         "qiita.study_processed_data spd USING (processed_data_id) "
+         "WHERE spd.study_id = %s")
+  ```
+  * Correct:
+  ```python
+  sql = """SELECT processed_data_status
+           FROM qiita.processed_data_status pds
+              JOIN qiita.processed_data pd USING (processed_data_status_id)
+              JOIN qiita.study_processed_data spd USING (processed_data_id)
+           WHERE spd.study_id = %s"""
+  sql = "SELECT * FROM qiita.qiita_user"
+  ```
+3. Use PEP8-style indentation for the SQL queries, as it will improve readability. Common SQL best-practices recommend to have a new line for the `SELECT`, `FROM`, `WHERE` and similar clauses:
+  * Wrong:
+  ```python
+  sql = """SELECT udt_name FROM information_schema.columns WHERE 
+           column_name = %s AND table_schema = 'qiita' AND (table_name = %s
+           OR table_name = %s)"""
+  ```
+  * Correct:
+  ```python
+  sql = """SELECT udt_name
+           FROM information_schema.columns
+           WHERE column_name = %s AND table_schema = 'qiita'
+               AND (table_name = %s OR table_name = %s)"""
+  ```
+4. Never, NEVER, use python string formatting to complete the SQL query parameters. Use the `sql_args` parameter from the transaction object. This is a strong recommendation from the psycopg2 developers to avoid SQL injection attacks (see [here](http://initd.org/psycopg/docs/usage.html#the-problem-with-the-query-parameters) for more information):
```

Comment:
Not blocking but I think it will be better:
see \[here\](http://initd.org/psycopg/docs/usage.html#the-problem-with-the-query-parameters) -> \[see\](http://initd.org/psycopg/docs/usage.html#the-problem-with-the-query-parameters)